app/admin/medics.rb

```diff
@@ -2,7 +2,9 @@
 
 # ActiveAdmin description for Medic entity
 ActiveAdmin.register Medic do
-  permit_params :name, :profession, :crm, :profession, :email, :phone
+  status_options = Patient.statuses.map { |s| [I18n.t("activerecord.attributes.shared.statuses.#{s[0]}"), s[1]] }
```

Comment:
e se a gente colocar isso aqui dentro de uma função para reaproveitar, aparentemente é o mesmo código nas duas classes de descrição de admin?

Reply:
Isso me ocorreu também e eu tentei compartilhar os parâmetros usando esse approach [aqui](https://stackoverflow.com/questions/24789006/using-activeadmins-dsl-with-shared-module-includes).

Mas eu não consegui fazer funcionar :worried:
Vou insistir de novo na idéia.

Reply:
Verdade, vi aqui que é meio tenso.

mas e se vc criar um arquivo helper na pasta:

`app/admin/helpers/enum_helper.rb`
```ruby
  module EnumHelper
    def self.translated_options(enum, property_name)
      enum.map { |key,value| [I18n.t("activerecord.attributes.shared.${property_name}.${key}"), value]
    end
  end
```
e ai vc troca por

```suggestion
  status_options = EnumHelper.translated_options(Patient.statuses, 'statuses')
```
Além disso ficou faltando você fazer o commit do arquivo de tradução

Reply:
Justíssimo, funcionou perfeitamente.

Adicionei o arquivo de tradução, tinha esquecido :sweat: